src/js/network-vis.js

```diff
@@ -220,7 +220,7 @@ const renderNetwork = (dataNodes, dataEdges) => {
       });
       nodes.update(oldNodes);
     }
-    if (oldClickedEdgeIds.length > 0) {
+    if (oldClickedEdgeIds && oldClickedEdgeIds.length > 0) {
```

Comment:
comments please

Reply:
Don't see a point for comments on null checks.